client/src/views/WeatherPage.vue

```diff
@@ -1,168 +1,38 @@
 <template>
-  <div id="app">
-    <main>
-      <div class="search-box">
-        <input
-          type="text"
-          class="search-bar"
-          placeholder="Search..."
-          v-model="query"
-          @keyup.enter="fetchWeather"
-        />
-      </div>
-      <div class="weather-wrap" v-if="typeof weather.main != 'undefined'">
-        <div class="location-box">
-          <div class="location">
-            {{ weather.name }}, {{ weather.sys.state }}
-            {{ weather.sys.country }}
-          </div>
-          <div class="date">{{ dateBuilder() }}</div>
-          <div class="weather-box">
-            <div class="temp">{{ Math.round(weather.main.temp) }} °F</div>
-            <div class="weather">{{ weather }}</div>
+  <div>
+      <div v-for="(weather, index) in weatherList" :key="weather.id">
+          <div v-if="index % 8  === 0">
```

Comment:
I recommend adding a comment specifying why this condition on line 4 exists for readability